### PR TITLE
Wording change for B2c

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
Forum discussion: [https://forum.worldcubeassociation.org/t/wording-change-for-b2c/3136](https://forum.worldcubeassociation.org/t/wording-change-for-b2c/3136)

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -435,7 +435,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - B2) Starting the attempt:
     - B2a) The judge resets the timer(s) as in [Regulation A3b](regulations:regulation:A3b) and indicates that they are ready for the competitor to start the attempt (e.g. placing the puzzle cover in front of the competitor, saying "READY", giving a thumbs-up signal). The competitor must start the attempt within one minute, else the competitor forfeits the attempt (DNS), at the discretion of the judge (partially superseded by [Regulation Y7d1](regulations:regulation:Y7d1)).
     - B2b) The competitor uses their fingers to touch the elevated sensor surfaces of the timer. The competitor's palms must be facing down, and located on the side of the timer that is closer to them. Penalty: time penalty (+2 seconds).
-    - B2c) The competitor must have no physical contact with the puzzle before the start of the attempt. Penalty: time penalty (+2 seconds).
+    - B2c) The competitor must have no physical contact with the puzzle cover before the start of the attempt. Penalty: time penalty (+2 seconds).
```
Comment (lgarron) [inline wca-regulations.md]: I'd suggest making it clear that this doesn't supersede A2c.
```suggestion
    - B2c) The competitor must have no physical contact with the puzzle or the puzzle cover before the start of the attempt. Penalty: time penalty (+2 seconds).
```

**Comment by xsrvmy** [comment]
Shouldn't it be "when starting the attempt" instead of "before the start of the attempt"?

**Comment by Ivan-Brigidano** [comment]
The [Regulation A4b1](https://www.worldcubeassociation.org/regulations/#A4b1) states that:

> A4b1) The competitor must have no physical contact with the puzzle while starting the solve. Penalty: time penalty (+2 seconds).

So I guess it make sense to rewrite the Regulation B2c to match the wording.

**Review by mirandapedro — APPROVED** [review]
LGTM!

**Pull request merged**